app/helpers/application_helper.rb

```diff
@@ -216,4 +216,8 @@ def organization_url(organization)
   def sanitized_referer(referer)
     URL.sanitized_referer(referer)
   end
+
+  def sanitize_and_decode(str)
+    HTMLEntities.new.decode(sanitize(str).to_str)
```

Comment:
Please use `to_s` here, `to_str` and similar methods are for Ruby's [implicit conversions](https://idiosyncratic-ruby.com/54-try-converting.html).